pkg/commands/models/commit.go

```diff
@@ -65,3 +65,7 @@ func (c *Commit) IsMerge() bool {
 func (c *Commit) IsTODO() bool {
 	return c.Action != ""
 }
+
+func IsHeadCommit(commits []*Commit, index int) bool {
+	return index == 0 || (!commits[index].IsTODO() && commits[index-1].IsTODO())
+}
```

Comment:
I wasn't sure if this is an appropriate place for such a helper function, or where else it could go.

Reply:
And the logic was wrong of course; fixed in 800c69e98fae.

Reply:
You know what would actually be the most accurate thing to do?

Check what `git` thinks the `HEAD` is, but without actually shelling out to `git`, i.e. just reading `.git/HEAD`. 

Thoughts?

Reply:
I'm not saying it's necessarily better, I'm just mostly thinking out loud, sorry.

I was in the process of checking open PRs so I stumbled upon the one I had regarding this, and this one, so I kinda responded to that comment here. In hindsight, it seems out of the blue, yeah.

All in all, I have nothing against the current approach! :)